web/src/models/Transaction.model.ts

```diff
@@ -0,0 +1,10 @@
+import {TRawTransaction, TTransaction} from 'types/Transaction.types';
+
+const Transaction = ({id = '', name = '', description = '', version = 1}: TRawTransaction): TTransaction => ({
```

Comment:
Dude check what @cescoferraro did here https://github.com/kubeshop/tracetest/pull/1320 for the model as well, so we don't have issues moving forward